### DISCUSSION TITLE
v1.1.0-0

### DISCUSSION THREAD

**Discussion by milo526** [milo526/homebridge-tuya-web#276] [General]
- fix: upgrade dependencies  37e2506
- feat: Improves handling of color temperature.  6177954
- feat: Improves handling of brightness.  2abebd3

https://github.com/milo526/homebridge-tuya-web/compare/v1.0.1...v1.1.0-0

<hr /><em>This discussion was created from the release <a href='https://github.com/milo526/homebridge-tuya-web/releases/tag/v1.1.0-0'>v1.1.0-0</a>.</em>

**Comment by cladmonitor** [comment]
I've tried the min and max brightness settings to work around the alerts, the alerts are fixed but the dimming responds on the various tuya devices sporadically. In all but one case scrolling the diming within homekit, the lights randomly choose brightness levels, some of them even cause the lights to just flicker. The one case that seems to work is a single lighbulb, the response though is still not correct. from 1-128 the bulb goes from dim to max brightness and from 129 - 255 the bulb doesn't respond. I'm no expert here but is this an issue with sending large decimal place numbers, why would you not round to the nearest full value?

**Comment by mattxjones** [comment]
Have also been having this issue on 1.0.2, have updated to 1.1.0 and will keep you posted on how it goes.

FYI the status alert previously:

[30/08/2021, 11:06:26] [@milo526/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number 392 exceeded maximum of 100. See https://git.io/JtMGR for more info.

**Reply by mattxjones** [reply]
Still happening, bastard.

[30/08/2021, 11:17:15] [@milo526/homebridge-tuya-web] This plugin generated a warning from the characteristic 'Brightness': characteristic was supplied illegal value: number 392 exceeded maximum of 100. See https://git.io/JtMGR for more info.

**Comment by adude007** [comment]
The brightness corrections work but the color temperature gives me errors no matter what I set. If I check on the Tuya IoT platform the temperature on their end is expecting a value between 0-1000. 0 is the highest Kelvin value.
